app/workers/hard_worker.rb

```diff
@@ -0,0 +1,14 @@
+class HardWorker
+  include Sidekiq::Worker
+
+  def perform(user)
+    @articles = []
+    User.last(10).each do |u|
```

Comment:
In Rails you have to be carefull with the each and the arrays. In this case you are iterating though little quantity of collections. But if there were many, you would be pushing lots of models into an array. This would alloc lots of memory. Another option would be doing all this in one huge sql query. But for now this works fine